src/pages/index.js

```diff
@@ -1,10 +1,20 @@
-import React from 'react';
+import React, { useEffect, useState } from 'react';
 import CommunityLogo from '../assets/Community-Learning.svg';
-
+import EventCard from '../components/event-card';
 import Layout from '../components/layout';
 import SEO from '../components/seo';
+import mockEvents from '../mock-data/mockEvents';
 
 function IndexPage() {
+  const [upcomingEvents, setUpcomingEvents] = useState([]);
+
+  useEffect(() => {
+    const getEvents = async () => {
+      setUpcomingEvents(mockEvents);
+    };
+    getEvents();
```

Comment:
I'd like your opinion on the expected behavior or placeholder elements while this async function is firing. We can talk about it later. Likely, we can handle that in a separate ticket.

Reply:
I modified line 15 to this, to simulate an AJAX call that takes 5 seconds (or fails entirely):
```js
    setTimeout(getEvents, 5000);
```

The events section stays empty while the call is loading. I believe that the typical behavior would include a loading spinner or a placeholder object while the user's waiting on the response. Should we break that out into a separate ticket, do you think?
![image](https://user-images.githubusercontent.com/51807366/116500974-51167b80-a875-11eb-918c-ac58aaa26ddc.png)

Reply:
Seems like a design concern. Maybe we can have some "skeleton UI" blank cards that get replaced with real event cards once they're loaded.

Reply:
Good point! I'll send that question straight to the design team.